### ISSUE TITLE
Choose a webmap to show in background

### ISSUE THREAD

**Issue by larsdevries** [Esri/Survey123Community#178]
It would be very nice if you can add a (ArcGIS Online) Web map or add Feature / Map Services from a server. In my case I have a set of objects that need to be inspected and which locations are already known. It would be nice of I can add them to the map.

**Comment by martinGIS** [comment]
this would be very important for us too - sometimes we need the users to 'see' information such as vineyard boundaries or rasters in the back before they can complete the survey

**Comment by ryanl44** [comment]
This feature would definitely be helpful for us as well, it would be nice for inspectors to be able to have an electronic copy of the locations they're looking for as opposed to having to carry around paper copies.

**Comment by RKnepper3** [comment]
Yes any overlays would be great: show flood mapping, utility provider, custom zones, etc.  Right now we are planning on using Survey123 for collection with a web map created for reference that contains all of the reference layers they could open up in Explorer.  It works but its a pain to switch back and forth between apps.
